cmake/modules/Findpmdk.cmake

```diff
@@ -0,0 +1,58 @@
+# - Find pmdk
+#
+# pmdk_INCLUDE_DIRS - Where to find pmdk headers
+# pmdk_LIBRARIES - List of libraries when using pmdk.
+# pmdk_FOUND - True if pmdk found.
+
+find_package(PkgConfig QUIET REQUIRED)
+
+# all pmdk libraries depend on pmem, so always find it
+set(pmdk_FIND_COMPONENTS ${pmdk_FIND_COMPONENTS} pmem)
+list(REMOVE_DUPLICATES pmdk_FIND_COMPONENTS)
+
+foreach(component ${pmdk_FIND_COMPONENTS})
+  set(pmdk_COMPONENTS pmem pmemobj)
+  list(FIND pmdk_COMPONENTS "${component}" found)
+  if(found EQUAL -1)
+  endif()
```

Comment:
There was a `message(FATAL_ERROR ...` here, why was it removed?

Reply:
sorry, I deleted this line by mistake when deleting the message I added after debugging.